week-1-practise/level-3/1-piping.js

```diff
@@ -12,6 +12,11 @@
 
 var startingValue = 5;
 
+function piping(x) {
+  return startingValue * 2 + 10;
```

Comment:
How can you modify this, so instead of `*` and `+` you use the functions provided below? Also, remember about the `format` method :)